ccf-dynamodb/cloudformation/dynamodb.yml

```diff
@@ -0,0 +1,56 @@
+AWSTemplateFormatVersion: "2010-09-09"
+Description: Template that creates a DynamoDB table for the Copyright Cleared Feed platform
+
+Parameters:
+  TagSystemCode:
+    Description: The system code for the environment
+    Type: String
+    Default: upp
+
+  TagTeamDL:
+    Description: Tag for the team distribution list
+    Type: String
+    AllowedPattern: ^([a-zA-Z0-9_\-\.]+)@([a-zA-Z0-9_\-\.]+)\.([a-zA-Z]{2,5})$
+    ConstraintDescription: There must be a valid email address for the TeamDL
+    Default: universal.publishing.platform@ft.com
+
+  TagEnvironmentType:
+    Description: Tag detail for the Environment
+    Type: String
+    AllowedValues:
+        - 'd'
+        - 't'
+        - 'p'
+
+  TagName:
+    Description: Tag detail for the Name
+    Type: String
+    Default: CCF Client Settings
+
+  EnvironmentName:
+    Description: An environment name that will be appended to resource names
+    Type: String
+
+Resources: 
+  CCFDynamoDB: 
+    Type: AWS::DynamoDB::Table
+    Properties: 
+      TableName: "ccf-client-settings"
+      AttributeDefinitions: 
+        - 
```

Comment:
Only a question - what's about those empty lines?

Reply:
Nothing special, this is the way it was formatted in the User Guide's [example](https://docs.aws.amazon.com/AWSCloudFormation/latest/UserGuide/quickref-dynamodb.html).
I will remove them in order to be consistent with the `Tags` section.